src/Commands/HydroponicsDrushCommands.php

```diff
@@ -44,53 +46,152 @@ public function hydroponics(
       \Drupal::state()->set('hydroponics.heartbeat', $currenttime);
     });
     $active_queues = \Drupal::config('strawberryfield.hydroponics_settings')->get('queues');
+
+    // Store timer per queue
     $done = [];
 
+    // Store child timeout timer
+    $timeout = [];
+
     //track when queue are empty for n cycles
     $idle = [];
 
+    // store data for each child process
+    $child = [];
+
+    // track queue items count
+    $items = [];
+
+    // Parameters (ToDO put on config page?)
+    $queue_check_time = 1;
+    $child_timeout_time = 30;
+    $max_child_for_queue = 2;
```

Comment:
To think about: some queues need to run in sequence. Some queues may allow parallel processing.